### PR TITLE
feat: moar graphs and function update

### PR DESCRIPTION
**Pull request by CloudBeard**
Included bar graph and stacked bar graph functions.

Updated color scheme to follow UX team decision. 

Modified javascript name to align better

Updated javascript to accept file path to oscal but default is for DUBBD.

### CODE REVIEW

**Review by MxNxPx — APPROVED** [review]
LGTM!

File: .gitignore [hunk 1]
```diff
@@ -22,3 +22,6 @@
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
 replay_pid*
+
+# mac stuff
+.DS_Store
```
Comment (mjnagel) [inline .gitignore:27]: I don't think I can comment on the actual file but looks like `.DS_Store` got committed somehow.

Comment (CloudBeard) [reply .gitignore:27]: I noticed that and removed it/added to gitignore but it looks like its still in the remote branch. Ill see if I can delete from remote.

Comment (CloudBeard) [reply .gitignore:27]: Noticed it locally.

**Review by mjnagel — APPROVED** [review]

**Pull request merged**